service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/deser/ExceptionMessage.java

```diff
@@ -37,4 +38,19 @@ public String toString() {
                 ", detail='" + detail + '\'' +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        ExceptionMessage that = (ExceptionMessage) o;
+        return Objects.equals(name, that.name) &&
+            Objects.equals(detail, that.detail);
+    }
+
+    @Override
+    public int hashCode() {
+
+        return Objects.hash(name, detail);
+    }
```

Comment:
Added to ease testing.

Reply:
Does this solve the third problem of this issue: https://github.com/lagom/lagom/issues/425 ?

Reply:
Ah! I missed that. Thanks!

Reply:
I updated the comment in #425